addons/mail/static/src/models/message/message.js

```diff
@@ -395,6 +395,14 @@ function factory(dependencies) {
             return this.isModeratedByCurrentPartner;
         }
 
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeHasUnfollowIcon() {
+            return this.originThread && this.originThread.isCurrentPartnerFollowing;
+        }
```

Comment:
I think this can simply be a related field instead, no need to do the compute manually.